.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 repos:
 - repo: https://github.com/astral-sh/ruff-pre-commit
   # Ruff version.
-  rev: v0.0.277
+  rev: v0.6.0
```

Comment:
I updated this to version 0.6.0 to have the same behaviour in pre-commit and during the github-actions-build.